build.psm1

```diff
@@ -813,8 +813,10 @@ function Start-PSPester {
         [string]$binDir = (Split-Path (New-PSOptions).Output),
         [string]$powershell = (Join-Path $binDir 'powershell'),
         [string]$Pester = ([IO.Path]::Combine($binDir, "Modules", "Pester")),
+        [Parameter(ParameterSetName='Unelevate',Mandatory=$true)]
```

Comment:
I'll file an issue to move the Unelevated tests to use the same mechanism to verify that the tests passed, which should get rid of this.